configure/Makefile

```diff
@@ -31,3 +32,13 @@ CFG += CONFIG_DATABASE_VERSION
 
 include $(TOP)/configure/RULES
 
+ifdef T_A
+
+install: $(TOP)/configure/os/CONFIG_TOOLCHAIN.Common.$(T_A)
+
+$(TOP)/configure/os/CONFIG_TOOLCHAIN.Common.$(T_A): toolchain.c
+	$(PREPROCESS.cpp)
+
+CLEANS += ../os/CONFIG_TOOLCHAIN.Common.$(T_A)
```

Comment:
It seems that `$(CLEANS)` isn't actually cleaned by `make distclean` and similar.  I'm thinking to switch to something like:

```make
ifndef T_A
realclean: clean_toolchain
clean: clean_toolchain

clean_toolchain:
	$(RM) $(wildcard os/CONFIG_TOOLCHAIN.Common.*)
endif
```